app/models/performance_statistics.rb

```diff
@@ -52,14 +50,18 @@ def application_form_query
     SQL
   end
 
-  def ended_without_success_sql
+  def form_is_unsubmitted_sql
+    "('unsubmitted' = ANY(ARRAY_AGG(ch.status)) OR '{NULL}' = ARRAY_AGG(DISTINCT ch.status))"
```

Comment:
Is this saying "any form with > 0 unsubmitted applications is considered to be unsubmitted", whereas it used to need to be all the applications?